bulma/dark-rtl.scss

```diff
@@ -0,0 +1,15 @@
+@import './dark/variables';
```

Comment:
there is a slight change in this file and the light as well in order to fix the `gap` problem.

Please update it to the latest form. Thanks

Reply:
ok i will update it